x-pack/plugins/security_solution/public/common/components/exceptions/edit_exception_modal/index.tsx

```diff
@@ -83,14 +88,18 @@ const ModalBodySection = styled.section`
 
 export const EditExceptionModal = memo(function EditExceptionModal({
   ruleName,
+  ruleId,
   ruleIndices,
   exceptionItem,
   exceptionListType,
   onCancel,
   onConfirm,
+  onRuleChange,
 }: EditExceptionModalProps) {
   const { http } = useKibana().services;
   const [comment, setComment] = useState('');
+  const { rule: maybeRule } = useRuleAsync(ruleId);
```

Comment:
We may be able to refactor these modals to have a common parent, since they both repeat a good amount of logic, right now just doubling logic in each file.